lib/remotr/test_helpers.rb

```diff
@@ -25,9 +25,9 @@ def redirect_httparty(method)
           case method
           when :post
             query_string = options[:query].to_query.present? ? "?#{options[:query].to_query}" : nil
-            send method, "#{url.path}#{query_string}", options[:body], options[:headers]
```

Comment:
For `post` calls, `body` should be sent as the `body` keyword param instead of `params`, cf. http://api.rubyonrails.org/classes/ActionController/TestCase/Behavior.html#method-i-get